src/dags/cmsa.py

```diff
@@ -226,8 +224,3 @@
     Example geosearch:
     https://api.data.amsterdam.nl/geosearch?datasets=cmsa/locatie&x=106434&y=488995&radius=10
 """
-if __name__ == "__main__":
-    from airflow.utils.state import State
-
-    dag.clear(dag_run_state=State.NONE)
-    dag.run()
```

Comment:
nice.

Reply:
Together with the `DebugExecutor` this allows you to run the module containing the DAG as a regular Python program. Full power of PyCharm's debugger at your disposal. No need to run a separate `airflow scheduler` process.